client/checkout/blocks/upe.js

```diff
@@ -45,7 +45,10 @@ registerPaymentMethod( {
 	ariaLabel: __( 'WooCommerce Payments', 'woocommerce-payments' ),
 	supports: {
 		showSavedCards: getConfig( 'isSavedCardsEnabled' ) ?? false,
-		showSaveOption: getConfig( 'isSavedCardsEnabled' ) ?? false,
+		showSaveOption:
+			( getConfig( 'isSavedCardsEnabled' ) &&
+				! getConfig( 'cartContainsSubscription' ) ) ??
+			false,
```

Comment:
Is the `?? false` bit here redundant? The `&&` operator should result in either true or false if either condition is falsey, which should be enough I think.

Reply:
> Is the ?? false bit here redundant?

Hi @chickenn00dle `getConfig` could technically return a `null` https://github.com/Automattic/woocommerce-payments/blob/develop/client/utils/checkout.js#L9, `null && null` would still be a `null`. I am not sure of the chances of these configs actually not being set though.

Reply:
Thanks for clarifying. This makes sense 👍